packages/suite-build/configs/base.webpack.config.ts

```diff
@@ -132,7 +132,7 @@ const config: webpack.Configuration = {
                     },
                 ],
             },
-            // Workers
+            // Workers TODO(karliatto): is this worker-loader really loading anything here?
```

Comment:
does it work without it?

Reply:
I think that if you remove it, it would break discovery on web

Reply:
Ok, so it removing actually works in web but it breaks discovery suite-desktop. That loader is the base used for suite-web and suite-desktop. So I updated the comment to https://github.com/trezor/trezor-suite/pull/14959/commits/4b3fd0276aea0b28a72f5fa3f72332cec35a736a